packages/core/src/modules/oob/repository/outOfBandRecordMetadataTypes.ts

```diff
@@ -1,5 +1,8 @@
+import type { InvitationType } from '../messages'
+
 export enum OutOfBandRecordMetadataKeys {
   RecipientRouting = '_internal/recipientRouting',
+  LegacyInvitation = '_internal/legacyInvitation',
```

Comment:
What is our policy on new internal metadata tags? It can be seen as a breaking change if someone, weirdly, uses the internal tags themselves. I think it is fine to keep as non-breaking, just pointing it out.

Reply:
I think metadata should be seen as there can be a lot of keys. anything that starts with `_internal` is internal and we can change them however we like.